src/main/java/com/yahoo/bullet/record/avro/LazyBulletAvro.java

```diff
@@ -37,7 +41,41 @@
     private byte[] serializedData;
 
     private static final long serialVersionUID = -5368363606317600282L;
-    private static final SpecificDatumWriter<BulletAvro> WRITER = new SpecificDatumWriter<>(BulletAvro.class);
```

Comment:
What was wrong with this?

Reply:
I mean I don't think there was a need to store that. We didn't for the reader, so why the writer. The constructors are pretty light. We can store both though if that's what you'd like.